de4dot.code/deobfuscators/Dotfuscator/StringDecrypter.cs

```diff
@@ -90,8 +92,21 @@ public class StringDecrypterInfo {
 					var ldci4 = instrs[i + 3];
 					if (!ldci4.IsLdcI4())
 						continue;
+					bool isPro = false;
```

Comment:
Have you verified this code with an old version? Seems like it will fail to decrypt strings in older versions since it continues. (I haven't tested the code yet, maybe you have)

Reply:
yes. I had tested some files.

Reply:
Yes ,I had tested some files.
and changed the codes .